index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+
+<head>
+  <title>Broadband Economic Development Map</title>
```

Comment:
I think I've read a few different titles now, is this the one you want?

Economic Development Map is in the readme.